src/integrations/feed/configs.ts

```diff
@@ -0,0 +1,80 @@
+import { FeedConfig, FeedConfigGenerator } from './types';
+import { feedToFilters } from '../../common';
+
+type Options = {
+  includeAllowedTags?: boolean;
+  includeBlockedTags?: boolean;
+  includeBlockedSources?: boolean;
+  includeSourceMemberships?: boolean;
+};
+
+function getDefaultConfig(
+  baseConfig: Partial<FeedConfig>,
+  userId: string | undefined,
+  feedId: string | undefined,
```

Comment:
The `baseConfig` has a property `user_id`, why did we end up having the second parameter as the `userId`? The same goes for the `feedId`, though understandable as we want to set a default value for it, if so, we should maybe move it as the last parameter and set the default value from there.

If some of the properties are not really expected from the `constructor` when setting the `FeedConfig`, we should maybe use `Omit` to make it straightforward that those values would only come from the generate function parameters.